src/content/en/updates/2017/10/devtools-release-notes.md

```diff
@@ -0,0 +1,145 @@
+
+{% include "web/tools/chrome-devtools/_shared/styles.html" %}
+
+# What's New In DevTools (Chrome 63) {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+Note: The video version of these release notes will be published around
+early-December 2017.
+
+Welcome back! New features coming to DevTools in Chrome 63 include:
+
+* [Multi-client remote debugging support](#multi-client).
+* [Workspaces 2.0](#workspaces).
+* [Simulate push notifications with custom data](#push).
+* [Trigger background sync events with custom tags](#sync).
+
+Note: You can check what version of Chrome you're running at
+`chrome://version`. Chrome auto-updates to a new major version about every 6
+weeks.
+
+## Multi-client remote debugging support {: #multi-client }
+
+If you've ever tried debugging an app from an IDE like VS Code or WebStorm,
+you've probably discovered that opening DevTools messes up your debug session.
+This issue also made it impossible to use DevTools to debug WebDriver tests.
+See the video below for an example of the issue in VS Code.
+
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="OJlOU6UecFU"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+As of Chrome 63, this problem is no more. DevTools now supports multiple
+remote debugging clients. Watch the video below to see an example of VS Code
+and DevTools in action, side-by-side.
+
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="tTWs0ZyXI-0"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+## Workspaces 2.0 {: #workspaces }
+
+Workspaces have been around for some time in DevTools. This feature enables
+you to use DevTools as your IDE. You make some changes to your source code
+within DevTools, and the changes persist to the local version of your
+project on your file system.
+
+Workspaces 2.0 builds off of 1.0, adding a more helpful UX and improved
+auto-mapping of transpiled code.
+This feature was originally scheduled to be released shortly after
+Chrome Developer Summit (CDS) 2016, but the team postponed it to sort out
+some issues.
+
+Check out the "Authoring" part (around 14:28) of the DevTools talk from
+CDS 2016 to see Workspaces 2.0 in action.
+
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="HF1luRD4Qmk"
+          data-autohide="1" data-showinfo="0" frameborder="0" 
+          data-start="868" allowfullscreen>
+  </iframe>
+</div>
+
+## Simulate push notifications with custom data {: #push }
+
+Simulating push notifications has been around for a while in DevTools,
+with one limitation: you couldn't send custom data. But with
+the new **Push** text box coming to the **Service Worker** pane in Chrome 63,
+now you can. Try it now:
```

Comment:
Reads a little funny. Following a similar style to background sync events below, how about something like this:

"Simulating push notifications has been around for awhile in DevTools, but you couldn't send custom data. The new push text box coming to the Service Worker pane in Chrome 63 lets you write something in a text box and send it as custom data:"